test/unit/plugins/rest_spec.js

```diff
@@ -646,4 +647,233 @@ describe("The Rest plugin", function () {
 			});
 		});
 	});
+
+	describe("updating a created instance", function () {
+
+		var server    = new Hapi.Server();
+		var instances = new InstanceAdapter();
+
+		before(function () {
+
+			server.connection();
+			return server.registerAsync({
+				register : Rest,
+				options  : {
+					instances : instances
+				}
+			});
+		});
+
+		describe("setting the status of a created instance to failed", function () {
+
+			var responseCode;
+			var updatedInstance;
+			var instanceID;
+			before(function () {
+
+				return instances.createInstance()
+				.then(function (instance) {
+					instanceID = instance.id;
+					return instances.getInstance({ id : instance.id });
+				})
+				.then(function (instance) {
+					updatedInstance = new Instance({
+						id       : instance.id,
+						ami      : instance.ami,
+						type     : instance.type,
+						state    : "failed",
+						uri      : instance.uri,
+						revision : 1
+					});
+					var request = new Request("PUT", "/v1/instances/" + instance.id).mime("application/json")
+					.payload({
+						ami      : instance.ami,
+						type     : instance.type,
+						uri      : instance.uri,
+						state    : "failed",
+						revision : 0
```

Comment:
Similar to the previous comment, I would expect this to come from the fetched instance rather than being hard-coded.